web/pgadmin/tools/sqleditor/static/js/components/sections/MainToolBar.jsx

```diff
@@ -56,6 +57,7 @@ export function MainToolBar({containerRef, onFilterClick, onManageMacros, onAddT
   const eventBus = useContext(QueryToolEventsContext);
   const queryToolCtx = useContext(QueryToolContext);
   const queryToolConnCtx = useContext(QueryToolConnectionContext);
+  const modalId = MODAL_DIALOGS.QT_CONFIRMATIONS + queryToolCtx.params.trans_id;
```

Comment:
Qt Modals are independent of other modal instances of the pgAdmin app. I don't think you'll need to pass a trans id.

Reply:
Done.